core/src/input/input.stories.tsx

```diff
@@ -1,18 +1,38 @@
 import { _Story } from "../_story";
 import { Input } from "./input";
+import { Button } from "../button/button";
+import { DivPx } from "../div/div";
+import { useState } from "react";
+import { Dialog } from "../dialog/dialog";
 
 export default {
 	title: "Components/Input",
 	component: Input,
 	argTypes: {
-		type: _Story.arg(null),
+		type: {
+			name: "type",
+			type: { name: "object", required: true },
+			defaultValue: "text",
+			control: {
+				type: "inline-radio",
+				options: {
+					text: "text",
+					number: "number",
+					email: "email",
+					password: "password",
+					url: "url",
+				},
+			},
+		},
```

Comment:
You don't need to address this in this PR, but a note here is that Storybook's argType definition is often complicated (like the one above), so usually I'd extract and handle it in _Story.arg(...). It's a "try-to-be-smart" function that produce the complicated code above from an argument type.

In this case, for example, it should be something as simple as _Story.arg(["text", "number", ... ])

But yeah, maybe another PR